Examples/UIExplorer/js/TextExample.ios.js

```diff
@@ -22,6 +22,9 @@
  */
 'use strict';
 
+const Platform = require('Platform');
+const isTVOS = (Platform && Platform.OS === 'ios' && Platform.SystemName === 'tvOS');
```

Comment:
Instead of adding a new property here, why not use `Platform.interfaceIdiom == 'tv'`? A null-check on Platform also doesn't seem necessary to me.